packages/webpack-cli/lib/bootstrap.js

```diff
@@ -20,39 +20,21 @@ const isCommandUsed = (commands) =>
         return process.argv.includes(cmd.name) || process.argv.includes(cmd.alias);
     });
 
-const resolveNegatedArgs = (args) => {
-    args._unknown.forEach((arg, idx) => {
-        if (arg.includes('--') || arg.includes('--no')) {
-            const argPair = arg.split('=');
-            const optName = arg.includes('--no') ? argPair[0].slice(5) : argPair[0].slice(2);
-
-            let argValue = arg.includes('--no') ? 'false' : argPair[1];
-            if (argValue === 'false') {
-                argValue = false;
-            } else if (argValue === 'true') {
-                argValue = true;
-            }
-            const cliFlag = core.find((opt) => opt.name === optName);
-            if (cliFlag) {
-                args[cliFlag.group][optName] = argValue;
-                args._all[optName] = argValue;
-                args._unknown[idx] = null;
-            }
-        }
-    });
-};
-
 async function runCLI(cli, commandIsUsed) {
     let args;
-    const helpFlagExists = isFlagPresent(process.argv, 'help');
-    const versionFlagExists = isFlagPresent(process.argv, 'version') || isFlagPresent(process.argv, '-v');
+    const runVersion = () => {
+        cli.runVersion(commandIsUsed);
+    };
+    const parsedArgs = argParser(core, process.argv, false, process.title, cli.runHelp, runVersion);
 
-    if (helpFlagExists) {
+    if (parsedArgs.unknownArgs.includes('help')) {
         cli.runHelp(process.argv);
-        return;
-    } else if (versionFlagExists) {
-        cli.runVersion(commandIsUsed);
-        return;
+        process.exit(0);
+    }
+
+    if (parsedArgs.unknownArgs.includes('version')) {
+        runVersion();
+        process.exit(0);
```

Comment:
Maybe we can using `help` (https://github.com/tj/commander.js/#helpcb and https://github.com/tj/commander.js/#addhelpcommand) for help, same for version? Yes we can do it in future

Reply:
Yes, just don't want to do it here because it will probably involve some significant rewriting of https://github.com/webpack/webpack-cli/blob/next/packages/webpack-cli/lib/groups/HelpGroup.js